### ISSUE TITLE
Problem with libpinmame iOS build

### ISSUE THREAD

**Issue by droscoe** [vpinball/pinmame#143]
This just popped up today. I doubt it's related to my code.  I've pushed several updates just to stay in sync with master, with no new changes of my own:
```
-- Downloading...
   dst='/Users/runner/work/pinmame/pinmame/build/Release/zlib-prefix/src/zlib-1.2.13.tar.gz'
   timeout='none'
   inactivity timeout='none'
-- Using src='https://zlib.net/zlib-1.2.13.tar.gz'
-- [download 0% complete]
CMake Error at zlib-stamp/download-zlib.cmake:170 (message):
  Each download failed!

    error: downloading 'https://zlib.net/zlib-1.2.13.tar.gz' failed
          status_code: 22
          status_string: "HTTP response code said error"
          log:
          --- LOG BEGIN ---
            Trying 85.187.148.2:443...

  Connected to zlib.net (85.187.148.2) port 443 (#0)

  ALPN: offers h2

  ALPN: offers http/1.1

  [CONN-0-0][CF-SSL] (304) (OUT), TLS handshake, Client hello (1):

  [313 bytes data]

  [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, Server hello (2):

  [122 bytes data]

  [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, Unknown (8):

  [25 bytes data]

  [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, Certificate (11):

  [4630 bytes data]

  [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, CERT verify (15):

  [264 bytes data]

  [CONN-0-0][CF-SSL] (304) (IN), TLS handshake, Finished (20):

  [52 bytes data]

  [CONN-0-0][CF-SSL] (304) (OUT), TLS handshake, Finished (20):

  [52 bytes data]

  SSL connection using TLSv1.3 / AEAD-AES2[56](https://github.com/vpinball/pinmame/actions/runs/5906940957/job/16023962485#step:4:57)-GCM-SHA384

  ALPN: server accepted http/1.1

  Server certificate:

   subject: CN=zlib.net
   start date: Aug 14 00:00:00 2023 GMT
   expire date: Nov 12 23:[59](https://github.com/vpinball/pinmame/actions/runs/5906940957/job/16023962485#step:4:60):59 2023 GMT
   subjectAltName: host "zlib.net" matched cert's "zlib.net"
   issuer: C=US; ST=TX; L=Houston; O=cPanel, Inc.; CN=cPanel, Inc. Certification Authority
   SSL certificate verify ok.

  GET /zlib-1.2.13.tar.gz HTTP/1.1

  Host: zlib.net

  User-Agent: curl/7.79.1

  Accept: */*

  

  Mark bundle as not supporting multiuse

  HTTP/1.1 404 Not Found

  Date: Fri, 18 Aug 2023 20:46:03 GMT

  Server: Apache

  Strict-Transport-Security: max-age=[63](https://github.com/vpinball/pinmame/actions/runs/5906940957/job/16023962485#step:4:64)0[72](https://github.com/vpinball/pinmame/actions/runs/5906940957/job/16023962485#step:4:73)000; includeSubDomains

  X-Frame-Options: SAMEORIGIN

  X-Content-Type-Options: nosniff

  Content-Length: 315

  Content-Type: text/html; charset=iso-8[85](https://github.com/vpinball/pinmame/actions/runs/5906940957/job/16023962485#step:4:86)9-1

  The requested URL returned error: 404

  Closing connection 0
```

**Comment by droscoe** [comment]
Tried it again this morning.. still no joy.

**Comment by jsm174** [comment]
If you look at the zlib homepage they released 1.3 yesterday, so just bad timing I guess. 

I'm hoping its as simple as updating the iOS cmake file to point to `https://zlib.net/zlib-1.3.tar.gz`

**Comment by droscoe** [comment]
Surprised the previous version is not accessible any longer.Sent from my iPhoneOn Aug 19, 2023, at 10:47, jsm174 ***@***.***> wrote:﻿
If you look at the zlib homepage they released 1.3 yesterday, so just bad timing I guess.
I'm hoping its as simple as updating the iOS cmake file to point to https://zlib.net/zlib-1.3.tar.gz

**Comment by droscoe** [comment]
> If you look at the zlib homepage they released 1.3 yesterday, so just bad timing I guess.
> 
> I'm hoping it's as simple as updating the iOS cmake file to point to `https://zlib.net/zlib-1.3.tar.gz`

Just updated on my PR branch and it does fix the problem.  Once my PR is merged, this will be fixed

**Comment by jsm174** [comment]
Fixed in https://github.com/vpinball/pinmame/pull/144
